donjo-app/models/Keuangan_model.php

```diff
@@ -258,6 +258,8 @@ public function realisasi_pendapatan_desa($id_keuangan_master)
 
   public function widget_keuangan()
   {
+    $CI =& get_instance();
```

Comment:
Bentuk seperti ini hanya diperlukan apabila berada di luar file CI. Kalau di model tidak perlu. 

Bisa langsung dan lebih jelas menggunakan `$this->load->model('keuangan_grafik_model');`

Reply:
Sudah saya pindah untuk fungsinya ke `Keuangan_grafik_model.php` untuk fungsi widgetnya, saya kirim di PR #2222.